digdag-docs/src/scheduling_workflow.rst

```diff
@@ -105,6 +105,40 @@ Setting an alert if a workflow doesn't finish within expected time
     +long_running_job:
       sh>: long_running_job.sh
 
+In sla: directive, you can select either the ``time`` or ``duration`` option.
+
+=============================== ================================================== ==========================
+Syntax                          Description                                        Example
+=============================== ================================================== ==========================
+time>: ``HH:MM:SS``             Set this job must be completed by ``HH:MM:SS``     time>: 12:30:00
+duration>: ``HH:MM:SS``         Set this job must be completed during ``HH:MM:SS`` duration>: 00:05:00
+=============================== ================================================== ==========================
+
+Options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This parameter supports fail: BOOLEAN and alert: BOOLEAN options. Setting fail: true makes the workflow failed. Setting alert: true sends an notification using above notification mechanism.
+
+* Setting ``fail: true`` makes the workflow failed.
```

Comment:
This option makes the workflow status to fail, but trailing tasks keep running. I understand that this behavior sometimes causes confuse users. I want to confirm this behavior as intentional or not.